api/wrappers.js

```diff
@@ -17,11 +17,79 @@
 'use strict';
 
 
+/**
+  Decorate the Admin facade class.
+
+  @param {Object} cls The auto-generated class.
+  @returns {Object} The decorated class.
+*/
+function wrapAdmin(cls) {
+
+  /**
+    RedirectInfo returns redirected host information for the model. In Juju it
+    always returns an error because the Juju controller does not multiplex
+    controllers.
+
+    This is overridden as the auto-generated version does not work with current
+    JAAS, as servers are returned slighty differently.
```

Comment:
"as the JAAS servers return slightly different" ?

Reply:
Improved the comment. and moved it inside the function, as it does not belong to the docstring.